base_layer/core/src/transactions/key_manager/inner.rs

```diff
@@ -1218,12 +1230,7 @@ where TBackend: KeyManagerBackend<PublicKey> + 'static
         self.crypto_factories
             .range_proof
             .verify_mask(output.commitment(), &private_key, value.into())?;
-        // Detect the branch we need to scan on for the key.
-        let branch = if output.is_coinbase() {
-            TransactionKeyManagerBranch::Coinbase.get_branch_key()
-        } else {
-            TransactionKeyManagerBranch::CommitmentMask.get_branch_key()
-        };
+        let branch = TransactionKeyManagerBranch::CommitmentMask.get_branch_key();
```

Comment:
Interesting, did we miss detected coinbase output keys? Otherwise, this is a breaking change to be reflected in the PR title and description as I see `TransactionKeyManagerBranch::Coinbase` has been removed.

Reply:
`TransactionKeyManagerBranch::Coinbase` is never used, except in tests